packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -71,9 +76,76 @@ export type CollapsibleProps = {
 } & EnsembleWidgetProps &
   HasItemTemplate;
 
+const Panel = ({
+  scope,
+  setTemplateItems,
+  template,
+}: {
+  scope: CustomScope;
+  template: CollapsibleItem;
+  setTemplateItems: React.Dispatch<React.SetStateAction<CollapsibleItem[]>>;
+}): null => {
+  const evaluated = useEvaluate({ ...template }, { context: scope });
+  const itemAddedRef = useRef(0);
+
+  const evaluatedItem = useMemo(() => {
+    const { key, label, children } = evaluated as {
+      [key: string]: Expression<string>;
+    };
+
+    const header = isString(label)
+      ? label
+      : EnsembleRuntime.render([unwrapWidget(label)]);
+
+    const content = isString(children)
+      ? children
+      : EnsembleRuntime.render([unwrapWidget(children)]);
+
+    return {
+      key,
+      label: header as Expression<string>,
+      children: content as Expression<string>,
```

Comment:
These should be ReactNodes.

Reply:
I got it, but what is this header and content are string only, the example I give above ?

Reply:
Strings are also React nodes.